### PR TITLE
Module 8

### PR DESCRIPTION
**Pull request by ConorMolloy**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,18 +6,20 @@ RUN python -m venv $VIRTUAL_ENV
...
 RUN pip install poetry
-EXPOSE 5000
 COPY poetry.lock pyproject.toml /app/
 RUN poetry install --no-root
 COPY . /app
 
 FROM base as production
-ENTRYPOINT poetry run gunicorn -w 4 -b 0.0.0.0:5000 run:app
+ENV PORT 5000
+ENTRYPOINT 'poetry run gunicorn -w 4 -b 0.0.0.0:$PORT "run:app"'
```
Comment (JackMead) [inline Dockerfile:15]: This command works for Heroku but not for docker locally - when I run it I get:
`/bin/sh: 1: poetry run gunicorn -w 4 -b 0.0.0.0:$PORT "run:app": not found`

Probably the easiest way to align those is to use a shell script as your entrypoint that calls this command

**Review by JackMead — CHANGES_REQUESTED** [review]
This looks good, I can see the app is on Heroku and deploying successfully through Travis. Before this is ready for merging, there are two things we should do:
* Update our docs - it would be nice to know when the pipeline runs, where the app is hosted, and where the image is stored (e.g. your dockerhub username is hidden so I can't find that image easily)
* Adjust our docker image for production so that it runs both in Heroku and in docker locally - currently that fails if run locally

File: Dockerfile (hunk 1 above)
Comment (ConorMolloy) [reply Dockerfile]: Should be fixed now, let me know if you are ok with the changes

**Review by JackMead — APPROVED** [review]
This looks good, thanks for making those changes and happy for you to merge when you're ready

**Pull request merged**